### PR TITLE
Fixed: deal with cases like FixedString(64).

### PR DESCRIPTION
**Pull request by 0x4ec7**
Extract base typename from types with additional argument.

### CODE REVIEW

**Comment by 0x4ec7** [comment]
@cpcloud, I've added test case for column types with additional argument.
1. DateTime with timezone, DateTime('UTC');
2. FixedString with length, FixedString(8)

**Comment by cpcloud** [comment]
@0x4ec7 Great thanks! The build failures are unrelated to your PR. We are tracking the root cause in #1511 and the linked JIRA.

File: ibis/clickhouse/tests/test_types.py [hunk 1]
```diff
@@ -14,3 +14,14 @@ def test_column_types(alltypes):
...
+
+
+def test_columns_types_with_additional_argument(con):
+    datetime_with_tz_sql = (
+        "SELECT toDateTime('2018-07-02 00:00:00', 'UTC') AS datetime_with_tz")
+    fixedstring_with_len_sql = (
+        "SELECT toFixedString('foo', 8) as fixedstring_with_len")
+    assert con.sql(datetime_with_tz_sql).execute()\
+              .datetime_with_tz.dtype.name == 'datetime64[ns]'
+    assert con.sql(fixedstring_with_len_sql).execute()\
```
Comment (cpcloud) [inline ibis/clickhouse/tests/test_types.py]: please make these into variables so you don't need the line break

File: ibis/clickhouse/tests/test_types.py [hunk 2]
```diff
@@ -14,3 +14,14 @@ def test_column_types(alltypes):
     assert df.float_col.dtype.name == 'float32'
     assert df.double_col.dtype.name == 'float64'
     assert df.timestamp_col.dtype.name == 'datetime64[ns]'
+
+
+def test_columns_types_with_additional_argument(con):
```
Comment (cpcloud) [inline ibis/clickhouse/tests/test_types.py]: It looks like this test is failing: https://circleci.com/gh/ibis-project/ibis/4360?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Comment (0x4ec7) [reply ibis/clickhouse/tests/test_types.py]: Yes, it is caused by clickhouse-server version difference. The version I used for testing is 1.1.54385, but docker-compose requires 1.1.54327. https://github.com/yandex/ClickHouse/blob/master/CHANGELOG.md#clickhouse-release-1154337-2018-01-18

Comment (cpcloud) [reply ibis/clickhouse/tests/test_types.py]: Should we update the docker image? I'm relatively ignorant about clickhouse, @kszucs is the resident expert on that.

Comment (kszucs) [reply ibis/clickhouse/tests/test_types.py]: IMHO we can just update the clickhouse image, they have frequent releases.

**Comment by cpcloud** [comment]
@0x4ec7 Can you rebase on top of master?

File: ibis/clickhouse/tests/test_types.py (hunk 2 above)
Comment (cpcloud) [reply ibis/clickhouse/tests/test_types.py]: I created #1513 to update the image. I'll get that working and then we can merge this PR.

**Comment by 0x4ec7** [comment]
@cpcloud Rebasing done.

**Comment by cpcloud** [comment]
@0x4ec7 Can you make these changes here, from this PR: https://github.com/ibis-project/ibis/pull/1513/files ? That PR updates the clickhouse image.

**Comment by cpcloud** [comment]
@0x4ec7 It looks like you pulled in old changes on top of yours. Can you fix up your commits to make sure that your changes are on top of current master?

**Comment by 0x4ec7** [comment]
@cpcloud I merged current master instead of rebasing onto it, hope not to mess up the branch.

**Comment by cpcloud** [comment]
Thanks @0x4ec7!

**Pull request closed without merging**